libraries/AP_Notify/Buzzer.h

```diff
@@ -19,14 +19,17 @@
 
 #include <AP_HAL/AP_HAL.h>
 
-#if CONFIG_HAL_BOARD == HAL_BOARD_VRBRAIN
+#ifndef BUZZER_PIN
+// better to define it in hal
+ #if CONFIG_HAL_BOARD == HAL_BOARD_VRBRAIN
  # define BUZZER_PIN    32
-#elif CONFIG_HAL_BOARD == HAL_BOARD_LINUX && CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_BBBMINI
+ #elif CONFIG_HAL_BOARD == HAL_BOARD_LINUX && CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_BBBMINI
```

Comment:
please don't add in unrelated whitespace in your patches

Reply:
You don't like #if/#endif indents? It helps to see logical structure

Reply:
All anothers fixed